exporter/clickhouselogsexporter/migrations/000001_init_db.up.sql

```diff
@@ -0,0 +1,59 @@
+CREATE TABLE IF NOT EXISTS logs (
```

Comment:
`logs` table will be created in `default` db?

Reply:
No